lessons/4_numbers/classwork.md

```diff
@@ -0,0 +1,46 @@
+# Lesson 4 Classwork
+
+The goal of todays class work is to make a sort of visual additional calculator using what we have
+learned in the first few lectures.
+
+- variables, strings and numbers
+- basic functions
+- document selectors like `getElementById`
+- document events like "change" and "click" via `addEventListener`
+- html element creation with `createElement` and `appendChild`
+
+> ![Lesson 4 Demo](./lesson_4_demo.png)
+> Styled Demo
+
+
+## Spec
+
+> ![Lesson 4 Demo Annotated](./lesson_4_demo_annotated.png)
+> Annotated Demo
+
+The app is comprised of four components:
+
+1. numbers list
+2. sum
+3. add button
+4. input
+
+It should work as follows:
+
+- numbers are entered into the input
+- upon clicking the add button the current number in the input is added to the numbers list and the
+  sum of the numbers is recalculated
+- after the number is added the input is cleared
+
+
+## Tips
+
+- it is possible to create this app without logic operators / control statements. No `if` statements
```

Comment:
Some of them don't know so far what if statement is

Reply:
Yeah this is a note for the students who do know or ones who have an idea about them. Letting them know we will focus on `if` statements later in the course.

Reply:
:+1: